juniper_official/vpn/evpn/evpn-check-mac-count-netconf.rule

```diff
@@ -0,0 +1,114 @@
+/*
+ *   This rule checks health of MAC count in evpn mac table and notify in case any of the health monitored field crosses threshold
+ */
+healthbot {
+    topic evpn {
+        description "This rule checks health of MAC count in evpn mac table and notify in case any of the health monitored field crosses threshold";
+        synopsis "MAC count check in evpn";
+        rule evpn-check-mac-count-netconf {
+            keys instance-name;
+            synopsis "MAC count in evpn mac table analyzer";
+            description "This rule checks health of MAC count in ethernet switching table and notify in case any of the health monitored field crosses threshold";
+            sensor evpn-mac-count {
+                synopsis "mac count netconf based iAgent sensor definition";
+                description "iAgent sensor defined in yml file using netconf rpc/cli command and defined fields using pyez tables&views to collect data from network device";
+                iAgent {
+                    file evpn-mac-count.yml;
```

Comment:
Is there no JTI sensor for this?

Reply:
JTI sensor is not available for some of the evpn kpis